specification/search/resource-manager/Microsoft.Search/stable/2015-08-19/search.json

```diff
@@ -274,6 +275,55 @@
             }
           }
         }
+      },
+      "post": {
+        "tags": [
+          "QueryKeys"
+        ],
+        "x-mspageable": {
+          "nextLinkName": null
+        },
+        "operationId": "QueryKeys_ListBySearchServicePost",
```

Comment:
That's not a great name for a public SDK method. Can we instead just remove the GET version of this from the Swagger and re-use the same name? Ideally the parameters wouldn't change on the client side so it wouldn't be a breaking change.

Reply:
I thought we get S360 issue if we still have something reachable without it being in the swagger?

Reply:
Define "reachable". If we remove it from our RP manifest and ensure it doesn't show up in the Operations API, is it no longer reachable? I realize that is not an option for this API version, but we should definitely do that in future API versions.

You can leave it in the Swagger, but I'd recommend finding a way to exclude that operation from codegen, and using the old name (which is a good name) for the new POST-based operation.

Reply:
Any ideas of how I can prevent it from codegen? I marked it with a deprecated tag, but not sure how that impacts codegen.

Reply:
AutoRest options are documented [here](https://github.com/Azure/autorest/tree/master/docs/extensions). If you can't find one that does what you want, you can always use the big hammer of adding regex rules to the [AutoRest config file](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/search/resource-manager/readme.md). If it comes to that, you can look in the data plane configs for examples.